16_woman_cancer.do

```diff
@@ -46,7 +46,11 @@ if inlist(name,"Jordan2007") {
 	replace w_mammogram=. if v012 < 20
 }
 
-
+if inlist(name,"Lesotho2009") {
+	replace w_papsmear=(s1012m==1)
+	replace w_papsmear=. if s1012m==.
```

Comment:
if inlist(name,"Lesotho2009") {
	replace w_papsmear=0 if !inlist(s1012l,9,.)
	replace w_papsmear=1 if s1012m==1 & inlist(s1012n,1,2) 
	replace w_papsmear=. if s1012n==8 // code . if uncertain about recall period 
    tab v012 if w_papsmear!=. /*DHS sample is women aged 15-49*/
	replace w_papsmear=. if inlist(v012,20,49)
}

Reply:
There's a filter question. 
![image](https://user-images.githubusercontent.com/63058112/113576262-3a219800-95ed-11eb-8f5b-9b6dcda7aed6.png)

Reply:
> Xian's code:
> 
> if inlist(name,"Lesotho2009") {
> replace w_papsmear=0 if !inlist(s1012l,9,.)
> replace w_papsmear=1 if s1012m==1 & inlist(s1012n,1,2)
> replace w_papsmear=. if s1012n==8 // code . if uncertain about recall period
> tab v012 if w_papsmear!=. /_DHS sample is women aged 15-49_/
> replace w_papsmear=. if inlist(v012,20,49)
> }

Question: why replace w_papsmear = . when age is within 20-49 range?
Overall, thanks for the example, very helpful. So we shall use indicator in combination with recall variables.